### ISSUE TITLE
Getting access denied to log directory to create files

### ISSUE THREAD

**Issue by artisticcheese** [Microsoft/IIS.Administration#110]
Hello,

Desptire the fact that `System` account has full access to `c:\inetpub\logs` directory, UI does not allow to create files or directories.
```
PS C:\inetpub\logs> get-acl ..\logs | select -ExpandProperty Access | where IdentityReference -eq "NT Authority\system"


FileSystemRights  : FullControl
AccessControlType : Allow
IdentityReference : NT AUTHORITY\SYSTEM
IsInherited       : True
InheritanceFlags  : None
PropagationFlags  : None

FileSystemRights  : 268435456
AccessControlType : Allow
IdentityReference : NT AUTHORITY\SYSTEM
IsInherited       : True
InheritanceFlags  : ContainerInherit, ObjectInherit
PropagationFlags  : InheritOnly

```

![image](https://user-images.githubusercontent.com/13176884/27994119-8a866736-647c-11e7-89c0-5a1deea4be08.png)

**Comment by jimmyca15** [comment]
The api administration only allows read access to the file system unless write privileges are granted in the appsettings.json file. Did you configure the api to allow for writing to the inetpub folder as shown here https://docs.microsoft.com/en-us/iis-administration/configuration/appsettings.json#files?

**Comment by artisticcheese** [comment]
I did not change anything out of default. So by default it will not allow access to system paths since it's not relying only ACLs but also on application stack for authorization.

**Comment by jimmyca15** [comment]
This is correct. To begin creating directories the appsettings.json [_files_](https://docs.microsoft.com/en-us/iis-administration/configuration/appsettings.json#files) section should be updated to allow write access.

Additionally, to expose other file system paths to the API, read access should be granted.

**Comment by jimmyca15** [comment]
Was this issue resolved along with #111?
